tests/DCPS/Compiler/typeobject_hash_consistency/run_test.pl

```diff
@@ -0,0 +1,24 @@
+eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
+    & eval 'exec perl -S $0 $argv:q'
+    if 0;
+
+use Env (DDS_ROOT);
+use lib "$DDS_ROOT/bin";
+
+use strict;
+
+my $gencmd = "$ENV{DDS_ROOT}/bin/opendds_idl -Sa -St sample.idl";
+system($gencmd);
+system("mv sampleTypeSupportImpl.cpp pass1");
```

Comment:
Don't assume that the system has a `mv` command.  Renaming can be handled by the perl script without spawning an external process.